src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/ProvideCorrectArgumentsToFormattingMethods.cs

```diff
@@ -114,13 +133,13 @@ public override void Initialize(AnalysisContext analysisContext)
                     int actualArgumentCount = intializer.ElementValues.Length;
                     if (actualArgumentCount != expectedStringFormatArgumentCount)
                     {
-                        operationContext.ReportDiagnostic(operationContext.Operation.Syntax.CreateDiagnostic(Rule));
+                        context.ReportDiagnostic(invocation.CreateDiagnostic(Rule));
                     }
                 }, OperationKind.Invocation);
             });
         }
 
-        private static int GetFormattingArguments(string format)
+        private static HashSet<int>? GetStringFormatItemIndexes(string format)
         {
             // code is from mscorlib
             // https://github.com/dotnet/coreclr/blob/bc146608854d1db9cdbcc0b08029a87754e12b49/src/mscorlib/src/System/Text/StringBuilder.cs#L1312
```

Comment:
Can we update this link to point to the current implementation [here](https://github.com/dotnet/runtime/blob/f1e131a4bef5bd5373a3dab65523851b54a94306/src/libraries/System.Private.CoreLib/src/System/Text/StringBuilder.cs#L1538) now that we've migrated repos?